streams/src/main/java/org/apache/kafka/streams/StreamsConfig.java

```diff
@@ -37,7 +37,6 @@
 import org.apache.kafka.streams.errors.LogAndFailExceptionHandler;
 import org.apache.kafka.streams.errors.ProductionExceptionHandler;
 import org.apache.kafka.streams.errors.StreamsException;
-import org.apache.kafka.streams.processor.DefaultPartitionGrouper;
```

Comment:
We cannot suppress deprecation warnings for imports -- hence, we need to remove imports of deprecated classed and use the fully qualified name in the code (applied to other classes in this PR, too).